src/dev/flang/ast/AstErrors.java

```diff
@@ -1524,6 +1524,12 @@ static void lossOfPrecision(SourcePosition pos, String _originalString, int _bas
       "Loss of precision for: " + _originalString,
       "Expected number given in base " + _base + " to fit into " + _type + " without loss of precision.");
   }
+
+  public static void argumentNamesNotDistinct(SourcePosition pos)
+  {
+    error(pos,
+      "Names of arguments must be distinct.", null);
```

Comment:
Be a bit more verbose. I usually add a detailed message and try to list the exact problems, in this case the names of all argument fields. Use `sbn(name)` in the text to get uniform color and quotation. Maybe also add the position of all arguments (by appending pos.show() for each name's position). 
Also, add a suggested fix at the end, e.g., "To solve this, rename argument `x` as `x1`.".

Reply:
Now it looks like this.
![image](https://user-images.githubusercontent.com/88769212/201298147-c35868db-c834-4d9e-b15d-5726f46aa250.png)

Unfortunately the parser does not give us the correct position of the arguments yet so I use the position of the feature.
I added a NYI so this can be fixed in another PR.

Reply:
You are using the Features of the arguments now, and they do have a `pos()` entry! 
Let me know if I should pull this PR into the repository (and you add the position in a new PR, or you want to update this.

Reply:
yes, but right now all the arguments have the position of the first arg as far as I can tell.

Reply:
@fridis Should I create an issue for this? This beeing all the args having the position of the first arg.